### DISCUSSION TITLE
strapi.query('model').search?

### DISCUSSION THREAD

**Discussion by ricar2ce** [strapi/strapi#6033] [Legacy Topics]
Reading the strapi documentation, in the queries api, I found the search method:

`strapi
  .query('restaurant')
  .search({ _q: 'my search query', _limit: 10, _start: 20 });`

but, what format or schema should it have?:  'my search query'

thanks you very much

**Comment by robertpiosik** [comment]
Well, it will use, for instance in MySQL [`MATCH AGAINST`](https://dev.mysql.com/doc/refman/8.0/en/fulltext-search.html) query on columns in specified model.

It will use [this function](https://github.com/strapi/strapi/blob/master/packages/strapi-connector-bookshelf/lib/queries.js#L649) for SQL.
